app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Users::SessionsController < Devise::SessionsController
+  # before_action :configure_sign_in_params, only: [:create]
+
+  # GET /resource/sign_in
+  # def new
+  #   super
+  # end
+
+  # POST /resource/sign_in
+  # def create
+  #   super
+  # end
+
+  # DELETE /resource/sign_out
+  # def destroy
+  #   super
+  # end
+
+  # protected
+
+  # If you have extra params to permit, append them to the sanitizer.
+  # def configure_sign_in_params
+  #   devise_parameter_sanitizer.permit(:sign_in, keys: [:attribute])
+  # end
```

Comment:
What is the purpose of this file, please? There is nothing inside.
Kindly remove it, please.

Reply:
I have to keep it because I'm not using Devise directly in my views. I've configured the User views without the need to create the `devise` folder. My RegistrationsController and SessionsController exists under Users which inherits from Devise so I need to keep the Users::SessionsController, otherwise the app breaks. I tried removing it and realized why it breaks